src/globus_sdk/services/auth/client_types/base.py

```diff
@@ -219,17 +230,19 @@ def oauth2_refresh_token(self, refresh_token, additional_params=None):
         :param refresh_token: A Globus Refresh Token as a string
         :type refresh_token: str
 
-        :param additional_params: A dict of extra params to encode in the refresh call.
-        :type additional_params: dict, optional
+        :param query_params: A dict of extra params to encode in the refresh call.
+        :type query_params: dict, optional
         """
         log.info("Executing token refresh; typically requires client credentials")
         form_data = {"refresh_token": refresh_token, "grant_type": "refresh_token"}
 
-        if additional_params:
-            form_data.update(additional_params)
+        if query_params:
+            form_data.update(query_params)
```

Comment:
Since this case is a request body, we shouldn't call it `query_params`.

What about `form_params` or `body_params`?